### PR TITLE
testing: updated docker-compose.yml (MINOR)

### PR DESCRIPTION
**Pull request by big-andy-coates**
# Description 

Commit adds utility `up` script to bring up docker images, wait for ksql to be responsive, and then load the cli.  `down` to bring everything down again.

`.env` file added with environment vars for setting docker version.

updated some outdated links too.

### Testing done 

non-functional change.

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

**Comment by big-andy-coates** [comment]
> LGTM, I'll cherry-pick to the new `0.9.x-ksqldb` branch -- should this also go into the earlier (live) branches?

The doc changes probably should

File: .env [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Sets environment variables used in docker-compose.yml
+
+# Set to specific version needed, e.g. 5.5.0
+#CP_VERSION=latest
```
Comment (vcrfxia) [inline .env:4]: Remove?

File: up [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+echo -e "## Bringing up contains:\n"
```
Comment (vcrfxia) [inline up:6]:
```suggestion
echo -e "## Bringing up containers:\n"
```

File: down [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+echo -e "## Bringing down contains:\n"
```
Comment (vcrfxia) [inline down:6]:
```suggestion
echo -e "## Bringing down containers:\n"
```

File: ksqldb-docker/README.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ To build a new image with local changes:
 
 1. Build docker images from local changes.
     ```
-    > mvn -Pdocker package -DskipTests -Dspotbugs.skip -Dcheckstyle.skip  -Ddockerfile.skip=false -Dskip.docker.build=false -Ddocker.upstream-tag=latest-ubi8 -Ddocker.tag=local.build  -Ddocker.upstream-registry=''
+    > mvn -Pdocker package -DskipTests -Dspotbugs.skip -Dcheckstyle.skip  -Ddockerfile.skip=false -Dskip.docker.build=false -Ddocker.upstream-tag=latest -Ddocker.tag=local.build  -Ddocker.upstream-registry=''
```
Comment (vcrfxia) [inline ksqldb-docker/README.md:16]: Hang on, why are we undoing this change? Builds with the latest Dockerfile should fail with this change. Do they not for you?

Comment (big-andy-coates) [reply ksqldb-docker/README.md:16]: My bad - unintentional change.

**Comment by big-andy-coates** [comment]
> Sorry, undoing my green tick since I didn't notice one of the changes. Docker builds should fail if the base image tag is `latest` rather than `latest-ubi8`.

No worries.  All updated.

Though next time feel free to 'approve assuming you fix the latest tag', and then I can make the change and merge.  :)

**Comment by vcrfxia** [comment]
> Though next time feel free to 'approve assuming you fix the latest tag', and then I can make the change and merge. :)

Yeah, sorry about that -- I normally do this but didn't see a way to undo my initial green tick and was worried it'd be confusing. I thought it'd be okay since this PR didn't seem to be blocking your other work.

**Comment by big-andy-coates** [comment]
No biggie at all :)

**Comment by big-andy-coates** [comment]
@JimGalasyn feel free to merge the doc changes where needed :D

**Comment by JimGalasyn** [comment]
@big-andy-coates I cherry-picked this PR to the 0.9.x-ksqldb branch. The .yml and script changes look like they'll work in earlier versions, so I'd like to cherry-pick them to the live branches too, along with the doc updates.

**Pull request merged**